src/Storage/Storage.Management/help/Update-AzStorageBlobServiceProperty.md

```diff
@@ -103,6 +105,41 @@ IsVersioningEnabled           : True
 
 This command enables Versioning on Blob service of a Storage account
 
+### Example 4: Update CORS rules
```

Comment:
Please also give the sample to clear the CORS rule.
To match the usage of each dataplane CORS rule cmdlets.

Reply:
Added